frontend/appflowy_flutter/packages/appflowy_editor/lib/src/render/image/image_upload_widget.dart

```diff
@@ -51,12 +56,12 @@ void _dismissImageUploadMenu() {
 }
 
 class ImageUploadMenu extends StatefulWidget {
-  const ImageUploadMenu({
-    Key? key,
-    required this.onSubmitted,
-    required this.onUpload,
-    this.editorState,
-  }) : super(key: key);
+  const ImageUploadMenu(
+      {Key? key,
+      required this.onSubmitted,
+      required this.onUpload,
+      this.editorState})
+      : super(key: key);
```

Comment:
```suggestion
  const ImageUploadMenu({
    super.key,
    required this.onSubmitted,
    required this.onUpload,
    this.editorState,
  });
```
Use new super initializer in constructors, honestly this might look stupid, but it actually removes one line missing to be tested, whenever this is done. It is much smarter and easier to read.